app/assets/stylesheets/common/base/about.scss

```
@@ -11,11 +19,23 @@
     border-bottom: 1px solid var(--primary-low);
     padding: 1em 1em;
     margin-bottom: 1em;
+    @media screen and (max-width: 750px) {
+      flex-direction: column;
+    }
```

If you'd like to use one of the existing breaking points you can do e.g.

```suggestion
    @include breakpoint(mobile-extra-large) {
      flex-direction: column;
    }
```

for 550px (or `tablet` for 768px)

---

Nice, thanks!